### ISSUE TITLE
V5 import css error

### ISSUE THREAD

**Issue by ruicky** [ant-design/ant-design-mobile#3885]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design-mobile/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://github.com/ant-design/ant-design-mobile](https://github.com/ant-design/ant-design-mobile)

### Steps to reproduce
when i  in `app.js`  import css file, like this:
`import 'antd-mobile-alpha/index.css'`

then `npm start` would throw error :
```
Error: Can't resolve './assets/slider-thumb.svg' in '/xxx/node_modules/antd-mobile-alpha'
```
PS: "xxx" is my foldar path.

The problem is probably here, the url address is not processed, and the relative path is used。

like ` url('./assets/slider-thumb.svg')`

```
.am-slider-thumb {
  touch-action: none;
  position: absolute;
  width: 22px;
  height: 22px;
  border-radius: 50%;
  left: 50%;
  top: 50%;
  transform: translate(-50%, -50%);
  background: #fff url('./assets/slider-thumb.svg') no-repeat center center;
  background-size: 10px 10px;
  box-shadow: 0 3px 5px 0 rgba(0, 0, 0, 0.12), 0 1.5px 3px -2px rgba(0, 0, 0, 0.25);
}
```

### What is expected?
can run success

### What is actually happening?
run error

| Environment | Info |
|---|---|
| antd | undefined |
| React | 17 |
| System | mac |
| Browser | Chrome |

**Comment by ruicky** [comment]
antd-mobie version is `5.0.0-alpha.8`

**Comment by awmleer** [comment]
`import 'antd-mobile-alpha/index.css'` 会导致相对路径找不到 assets 文件，我修复一下

**Comment by awmleer** [comment]
Fixed via 1e9be33fa

**Comment by awmleer** [comment]
https://github.com/ant-design/ant-design-mobile/releases/tag/v5.0.0-alpha.9
